lib/src/support.dart

```diff
@@ -55,12 +55,12 @@ class Feature {
   static final Feature touchEvents = Feature(
       'touch',
       window.navigator.maxTouchPoints != null
-          ? window.navigator.maxTouchPoints > 0
+          ? window.navigator.maxTouchPoints! > 0
```

Comment:
Interesting. I'm surprised the new [type promotion on null checks](https://dart.dev/null-safety/understanding-null-safety#type-promotion-on-null-checks) flow analysis doesn't make this unnecessary.

Reply:
I bet if it were in a regular if block it would. The ternary operator might not handle it right.

Reply:
FYI I was planning on leaving this `!`